definitions/divorce/json/CaseEvent/CaseEvent-bailiff-nonprod.json

```diff
@@ -98,5 +98,16 @@
     "SecurityClassification": "Public",
     "ShowSummary": "Y",
     "EndButtonLabel": "Submit AOS Response"
+  },
+  {
+    "LiveFrom": "08/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "ID": "aosNotReceivedForBailiff",
+    "Name": "Move to awaiting DN",
```

Comment:
Although this is what the event does, I'd call it what it represents (i.e. AOS not received for Bailiff, if that name makes sense, or just AOS not received (Bailiff)).